### ISSUE TITLE
[Issue]: library.db  library.db-journal    Why do these two files keep writing to the hard disk? How do I turn off writing?

### ISSUE THREAD

**Issue by lijiazhiren** [jellyfin/jellyfin#8604]
### Please describe your bug

Every few seconds, data less than 1m is written to the hard disk. The sound of the hard disk is a bit loud. Where can I turn off this behavior?

### Jellyfin Version

10.8.0

### Environment

```markdown
- OS:
- Virtualization:docker
- Clients:
- Browser:
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by crobibero** [comment]
You cant disable writing to the database. This is not a bug.

**Comment by felixscheinost** [comment]
Hi, I found this issue as I noticed as well that Jellyfin excessively writes to this file.

I am using a Intel based Synology Diskstation NAS with a 4 core CPU and 10 GB of RAM which shouldn't be too underpowered. 

In the system monitor I can see a continous write of 1-2 MB/s which I find quite excessive. It's roughly double the read speed, which I would've expected to dominate. That comes from 1 non-transcoding stream.

Am I assuming correctly that this high load comes from writing playback progress? If yes, is playback progress maybe written too often? TO reach 1-2 MB/s I assume you would have to do quite a lot of changes to the DB per second?
